.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+    "presets": [
+        "es2015",
```

Comment:
There's a couple of things that make me unsure about making this change:

**Version Ranges**
We will no longer explicitly control the version of JS that we target. New releases of browsers could change our outputted bundle between our releases without us realising. This is possibly ok if we're informing users of the version range in our docs, but see point below. However, it's true that the environment plugin allows us to specify specific versions, so perhaps this isn't a problem.

**Browser Support: Versions vs Features**
We recently discussed that informing users of the js/browser features that we require was preferred over stating browser version numbers (see [Required Browser Features)](https://github.com/ScottLogic/finput#required-browser-features). With that in mind I prefer seeing exactly which version of JS we're targeting rather than which browser versions.